packages/itwin/ec3-widget/src/components/Templates.tsx

```diff
@@ -35,32 +35,37 @@ export const Templates = ({ onClickCreate, onClickTemplateTitle, onExportResult
   const [modalIsOpen, openModal] = useState(false);
   const load = useCallback(async () => {
     setIsLoading(true);
-    if (iTwinId) {
-      const accessToken = await getAccessToken();
-      const templatesResponse = await configClient.getConfigurations(accessToken, iTwinId);
-      const configurations: Configuration[] = templatesResponse.map((x) => {
-        return {
-          displayName: x.displayName,
-          description: x.description ?? "",
-          id: x.id,
-          labels: [],
-          reportId: x._links.report.href.split("/reports/")[1],
-        };
-      });
-      setTemplates(configurations);
-    } else {
-      toaster.negative("Invalid iTwinId.");
+    try {
+      if (iTwinId) {
+        const accessToken = await getAccessToken();
+        const templatesResponse = await configClient.getConfigurations(accessToken, iTwinId);
+        const configurations: Configuration[] = templatesResponse.map((x) => {
+          return {
+            displayName: x.displayName,
+            description: x.description ?? "",
+            id: x.id,
+            labels: [],
+            reportId: x._links.report.href.split("/reports/")[1],
+          };
+        });
+        setTemplates(configurations);
+      } else {
+        toaster.negative("Invalid iTwinId.");
```

Comment:
I wonder if this should also be translated, since toasts are user facing.

Reply:
To be honest, this is a useless check. I removed it.